src/pymor/parallel/ipython.py

```diff
@@ -0,0 +1,260 @@
+# This file is part of the pyMOR project (http://www.pymor.org).
+# Copyright Holders: Rene Milk, Stephan Rave, Felix Schindler
+# License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+
+from itertools import izip, chain
+import os
+import time
+import weakref
+
+
+try:
+    from ipyparallel import Client, TimeoutError
+    HAVE_IPYTHON = True
+except ImportError:
+    try:
+        from IPython.parallel import Client, TimeoutError
+        HAVE_IPYTHON = True
+    except ImportError:
+        HAVE_IPYTHON = False
```

Comment:
It seems like `HAVE_IPYTHON` is not used within this file. Do we require it somewhere else?